src/api/courses/mock/courses.ts

```diff
@@ -10,6 +10,7 @@ const courses: CoursesModel = {
       short_description:
         'Поисково-спасательная работа, следовая работа, а так же поиск тел погибших с помощью собак.',
       lessons_count: 144,
+      start_date: '2023-20-01',
```

Comment:
Приходит в формате YYYY-MM-DD
```suggestion
      start_date: '2023-01-20',
```